src/NuGet.Core/NuGet.Commands/PublicAPI/net472/PublicAPI.Unshipped.txt

```diff
@@ -0,0 +1 @@
+static NuGet.Commands.UnexpectedDependencyMessages.GetProjectDependenciesMissingVersion(NuGet.ProjectModel.PackageSpec project) -> System.Collections.Generic.IEnumerable<NuGet.Common.RestoreLogMessage>
```

Comment:
Referring to https://github.com/NuGet/NuGet.Client/blob/dev/docs/nuget-sdk.md#changes-to-apis again. 

Does this API need to be public? 

This is a restore specific API, so while we can't change the past by making UnexpectedDependencyMessages public, we can certainly avoid future problems by simply keeping this API internal.

Reply:
Because I use this function in the tests it complains that it needs to be public, I tried using protected but complains that `UnexpectedDependencyMessages` is static and cannot use protected

Reply:
You can make it `internal`, not protected.

Reply:
We have internalsVisibleTo in our tests, so we should be able to test internal methods.